packages/remark-plugins/plugins/anchor-links/index.js

```diff
@@ -56,22 +56,23 @@ function processHeading(node, compatibilitySlug, links, headings) {
     .substring(level + 1)
     .replace(/<\/?[^>]*>/g, '') // Strip html
     .replace(/\(\(#.*?\)\)/g, '') // Strip anchor link aliases
+    .replace(/»/g, '') // Safeguard against double-running this plugin
```

Comment:
This ensures that even if the anchor-links plugin runs twice, the extracted headings have the correct text.

---

For context: we were already double-applying this plugin in `dev-portal`. To make things a little more complicated, in `dev-portal` for `/docs` we use `@hashicorp/react-docs-page`, which applies the plugin initially using `platform-markdown-utils` defaults (which itself used `@hashicorp/remark-plugins` up till now, but in this PR has been updated to use `@hashicorp/platform-remark-plugins`). In order to fix the anchor-links issue _and_ remove the duplicate plugin call, we'd need to ensure `docs-page` is resolving to the pre-release of `@hashicorp/platform-remark-plugins` publishing from this PR.

I started down the path of trying to do the dependency dance necessary for that, but couldn't get it to work. Maybe a combination of the relatively complex dependency chain, and quirks in the `changeset` pre-release process (not sure if adjacent packages resolve to pre-releases?).